modules/instrain/profile/main.nf

```diff
@@ -0,0 +1,46 @@
+process INSTRAIN_PROFILE {
+    tag "$meta.id"
+    label 'process_high'
+    label 'error_retry'
+    label 'process_long'
+
+    conda (params.enable_conda ? "bioconda::instrain=1.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/instrain%3A1.6.1--pyhdfd78af_0':
```

Comment:
```suggestion
        'https://depot.galaxyproject.org/singularity/instrain:1.6.1--pyhdfd78af_0':
```

Shouldn't this be a `:`? I assume actually it's equivalent but better to have the same as other modules/container IDs in this regard